Packs/ServiceNow/Integrations/ServiceNow/ServiceNow.yml

```diff
@@ -4138,10 +4138,129 @@ script:
+      defaultValue: '10'
+      description: Query results limit
+      isArray: false
+      name: limit
+      required: true
+      secret: false
+    deprecated: false
+    description: Query the sc_cat_item table in ServiceNow
+    execution: false
+    name: servicenow-items-query
+    outputs:
+    - contextPath: ServiceNow.CatalogItem.ID
+      description: Catalog item sys_id
+      type: String
+    - contextPath: ServiceNow.CatalogItem.Name
+      description: Catalog item name
+      type: String
+    - contextPath: ServiceNow.CatalogItem.Description
+      description: Catalog item description
+      type: String
+    - contextPath: ServiceNow.CatalogItem.Price
+      description: Catalog item price
+      type: Number
+  - arguments:
+    - default: false
+      description: Catalog item sys_id
+      isArray: false
+      name: sys-id
+      required: true
+      secret: false
+    deprecated: false
+    description: Retrieve catalog item information by sys_id
+    execution: false
+    name: servicenow-item-details-get
+    outputs:
+    - contextPath: ServiceNow.CatalogItem.ID
+      description: Catalog item sys_id
+      type: String
+    - contextPath: ServiceNow.CatalogItem.Name
+      description: Catalog item name
+      type: String
+    - contextPath: ServiceNow.CatalogItem.Description
+      description: Catalog item description
+      type: String
+    - contextPath: ServiceNow.CatalogItem.Price
+      description: Catalog item price
+      type: Number
+  - arguments:
+    - default: false
+      description: Catalog item sys_id
+      isArray: false
+      name: sys-id
+      required: true
+      secret: false
+    - default: false
+      description: Quantity of the item to order
+      isArray: false
+      name: quantity
+      required: true
+      secret: false
+    - default: false
+      description: If there are mandatory variables defined for the item, they must
+        be passed to the endpoint. For example, var1=value1,var2=value2.
+      isArray: false
+      name: variables
+      required: false
+      secret: false
+    deprecated: false
+    description: Orders the specified catalog item
+    execution: false
+    name: servicenow-item-order-create
+    outputs:
+    - contextPath: ServiceNow.OrderRequest.ID
+      description: Generated request sys_id
+      type: String
+    - contextPath: ServiceNow.OrderRequest.RequestNumber
+      description: Number of the generated request.
+      type: String
+  - arguments:
+    - default: false
+      description: Incident sys_id
+      isArray: false
+      name: incident-id
+      required: true
+      secret: false
+    - default: false
+      description: tag sys_id
+      isArray: false
+      name: tag-id
+      required: true
+      secret: false
+    - default: false
+      description: tag title
+      isArray: false
+      name: title
+      required: true
+      secret: false
+    deprecated: false
+    description: Add tag to specific incident
```

Comment:
This might confuse customers, as they will not see the tag in the incident. We need to add an explanation that it will show up in the Tags page.